### PR TITLE
Circle CI integration

### PR DESCRIPTION
**Pull request by bergundy**

### CODE REVIEW

File: src/test/test_lib.ts [hunk 1]
```diff
@@ -28,11 +23,11 @@ test('createClassValidator creates an ajv ValidateFunction for each method signa
     },
   };
   const validators = createClassValidator(schema, 'Foo', 'params');
-  expect(validators.hello({ name: 'heh' })).to.be.true;
-  expect(validators.hello({})).to.be.false;
+  t.true(validators.hello({ name: 'heh' }));
```
Comment (arielshaqed) [inline src/test/test_lib.ts:26]: `think(Reflect.selfReference()).that.tHis.is(more(readable))`

**Review by arielshaqed — APPROVED** [review]
Cool!

A browser inside a test inside a node process inside a docker container inside a hosted CI service inside another cloud?  We need to dig deeper.

I'm not qualified to comment on the browser testing changes, but given that it passes tests I like it!

File: src/test/test_lib.ts (hunk 1 above)
Comment (bergundy) [reply src/test/test_lib.ts:26]: lolz

**Comment by bergundy** [comment]
Thanks!

Yeah, the browser test is weird but I can't think of a better way to test it.

**Pull request merged**